docs/docs/Getting Started/admin_ui.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Using the Console
```

Comment:
Not sure if "Console" is confused with Terminal. I guess AWS calls it a console? Could see this being "Admin" or just "UI"?